src/Symfony/Component/Ldap/Entry.php

```diff
@@ -38,13 +45,21 @@ public function getDn()
     /**
      * Returns whether an attribute exists.
      *
-     * @param string $name The name of the attribute
+     * @param string $name          The name of the attribute
+     * @param bool   $caseSensitive Whether the check should be case-sensitive
      *
      * @return bool
      */
     public function hasAttribute($name)
```

Comment:
Please the new parameter as a comment already, as suggested in my previous review. This way, we don't forget to add it in 6.0.